config/default.yaml

```diff
@@ -140,6 +140,8 @@ bookends:
     # Plugins for build setup
     setup:
         - scm
+    teardown:
+        - screwdriver-artifact-bookend
```

Comment:
I think because of this: https://github.com/screwdriver-cd/screwdriver/blob/master/bin/server#L39-L52

scm is actually `screwdriver-scm-${scmConfig.plugin}`, depending on what users configure as their scm plugin.